src/index.html

```diff
@@ -7,12 +8,69 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The href attribute should link to a CSS file, not an SCSS file. SCSS files need to be compiled into CSS before they can be used in an HTML file. Please ensure that the correct CSS file is linked here.

Reply:
What i should do?